targets/TARGET_STM/can_api.c

```diff
@@ -646,8 +646,12 @@ void can_irq_set(can_t *obj, CanIrqType type, uint32_t enable)
 #include <string.h>
 #include <inttypes.h>
 
+#define ENABLED true
+#define DISABLED false
+
 static uint32_t can_irq_ids[CAN_NUM] = {0};
 static can_irq_handler irq_handler;
+static uint32_t rx_irq_status = DISABLED;
```

Comment:
Does this not need to be either an array `[CAN_NUM]` or a flag inside the `can_t` object? Probably the former, but I'm not quite sure how this is handling multiple instances.